src/kayenta/report/detail/multipleResultsTable.tsx

```diff
@@ -34,16 +36,20 @@ const MultipleResultsTable = ({
 
   let columns: ITableColumn<ICanaryAnalysisResult>[] = tagKeys.map(key => ({
     label: key,
-    width: 1,
+    width: 5,
```

Comment:
i'm not sure what the intended behavior here is around having multiple grouping tags, but either way it kind of seems like the right-rail metadata belongs as a fixed width element anchored to the right with however many columns filling the remaining space. don't have a working example of multiple groupings so will leave that as an exercise for someone else.